modules/serverless-blue-green-deploy.adoc

```diff
@@ -0,0 +1,112 @@
+[id="serverless-blue-green-deploy_{context}"]
+= Routing and managing traffic by using a blue-green deployment strategy
+
+You can safely reroute traffic from a live version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
+
+.Procedure
+
+. Create and deploy an app as a Knative service.
+. Find the name of the first revision that was created when you deployed the service, by viewing the output from the following command:
++
+[source,terminal]
+----
+$ oc get configurations <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
+. Add the following YAML to the service `spec` to send inbound traffic to the revision:
++
+.Example Route
+[source,yaml]
+----
+...
+spec:
+  traffic:
+    - revisionName: <first_revision_name>
+      percent: 100 # All traffic goes to this revision
+...
+----
+. Verify that you can view your app at the URL output you get from running the following command:
++
+[source,terminal]
+----
+$ oc get route <route_name>
```

Comment:
Stray occurence of route. This can be `oc get service` to stick to that abstraction.

Reply:
earlier we use `ksvc` and here it would be `service` - can we choose one of these maybe for use with `oc` and stick with it, or is there a reason it'd be different and not `ksvc` here too?

Reply:
`oc get service` won't work, because the K8s Service will be picked then.

Reply:
opps, sorry. Yeah I was wrong above :joy: it should be ksvc.

Reply:
Thanks 🙂 I'll update them